From 6d63302d8e7d35c4549c49b97c8667203bc22428 Mon Sep 17 00:00:00 2001
From: Dan Voiculeasa <dan.voiculeasa@windriver.com>
Date: Tue, 14 Jun 2022 16:01:56 +0300
Subject: [PATCH 2/8] Revert "Add "labels" to Armada deployment"

This reverts commit eb2e87d32b2b9c9853deb70ed2e7029380ef0e16.
Updating the sources from 7ef4b8643b5ec5216a8f6726841e156c0aa54a1a to
ddbdd7256c20f138737f6cbd772312f7a19f58b8.
This revert is here just to reduce the need for testing. Keeping the
helm charts without changes between 7ef4b8..ddbdd7.

Signed-off-by: Dan Voiculeasa <dan.voiculeasa@windriver.com>
---
 charts/armada/templates/deployment-api.yaml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/charts/armada/templates/deployment-api.yaml b/charts/armada/templates/deployment-api.yaml
index f1395ef..562e3d0 100644
--- a/charts/armada/templates/deployment-api.yaml
+++ b/charts/armada/templates/deployment-api.yaml
@@ -92,8 +92,6 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: armada-api
-  labels:
-{{ $labels | indent 4 }}
   annotations:
 {{ tuple $envAll | include "helm-toolkit.snippets.release_uuid" | indent 4 }}
 spec:
-- 
2.34.1

